downstream/modules/platform/proc-attaching-subscriptions.adoc

```diff
@@ -0,0 +1,79 @@
+// emurtoug removed this assembly from the Planning guide to avoid duplication of subscription content added to Access management and authentication
+
+[id="proc-attaching-subscriptions"]
+
+= Attaching your {PlatformName} subscription
+
+[role="_abstract"]
+You *must* have valid subscriptions attached on all nodes before installing {PlatformName}. Attaching your {PlatformNameShort} subscription provides access to subscription-only resources necessary to proceed with the installation.
+
+[NOTE]
+====
+Attaching a subscription is unnecessary if you have enabled Simple Content Access Mode on your Red Hat account. Once enabled, you will need to register your systems to either Red Hat Subscription Management (RHSM) or Satellite before installing the {PlatformNameShort}. For more information, see link:https://access.redhat.com/articles/simple-content-access[Simple Content Access].
+====
+
+.Procedure
+
+. Obtain the `pool_id` for your {PlatformName} subscription:
++
+-----
+# subscription-manager list --available --all | grep "Ansible Automation Platform" -B 3 -A 6
+-----
++
+[NOTE]
+====
+Do not use MCT4022 as a `pool_id` for your subscription because it can cause {PlatformNameShort} subscription attachment to fail.
+====
++
+.Example
+An example output of the `*subsciption-manager list*` command. Obtain the `pool_id` as seen in the `Pool ID:` section:
++
+-----
+Subscription Name: Red Hat Ansible Automation, Premium (5000 Managed Nodes)
+  Provides: Red Hat Ansible Engine
+  Red Hat Ansible Automation Platform
+  SKU: MCT3695
+  Contract: ````
+  Pool ID: <pool_id>
+  Provides Management: No
+  Available: 4999
+  Suggested: 1
+-----
++
+. Attach the subscription:
++
+-----
+# subscription-manager attach --pool=<pool_id>
+-----
++
+You have now attached your {PlatformName} subscriptions to all nodes.
++
+. To remove this subscription, enter the following command:
++
+-----
+# subscription-manager remove --pool=<pool_id>
+-----
+
+.Verification
+
+* Verify the subscription was successfully attached:
+
+-----
+# subscription-manager list --consumed
+-----
+
+.Troubleshooting
+
+* If you are unable to locate certain packages that came bundled with the {PlatformNameShort} installer, or if you are seeing a `_Repositories disabled by configuration_` message, try enabling the repository by using the command:
++
+{PlatformName} {PlatformVers} for RHEL 8
```

Comment:
Is the text '{PlatformName} {PlatformVers} for RHEL 8' intentional? Seems a little out of place to me.

Reply:
Yes, this is intentional. :D